File: src/main/resources/static/js/controller/quoteController.js
angular.module('quoteMachine')
    .controller('quoteController', ['$scope', 'QuoteService', function($scope, quoteService) {

        $scope.getNewQuote = function() {
            init();
        };

        var init = function(){
            quoteService.quotes.findOne(function(data){
                $scope.quote = data;
            });
        };
        init();
    }]);

File: src/main/java/br/com/bytecraft/controller/QuoteController.java
package br.com.bytecraft.controller;

import br.com.bytecraft.model.Quote;
import br.com.bytecraft.service.QuoteService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(value = "/")
public class QuoteController {

    @Autowired
    private QuoteService service;

    @RequestMapping(value = "quotes", method = RequestMethod.GET)
    public Quote findOne() {
        return service.getRandomQuote();
    }

}
